hpc_launcher/schedulers/__init__.py

```diff
@@ -30,3 +30,16 @@ def get_schedulers():
         "lsf": LSFScheduler,
         "LSFScheduler": LSFScheduler,
     }
+
+def parse_env_list(e: tuple[str,...]) -> str:
```

Comment:
If you use `*e` here you would not need all the extra parentheses everywhere else (and could pass in `parse_env_list(*e)`)